CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+2.0.0-alpha0 / TBD
```

Comment:
no need to use the `-alpha` in changelog. The CI will automatically add beta tags to releases.